package.json

```diff
@@ -2,5 +2,17 @@
   "name": "uptempo-web",
   "version": "1.0.0",
   "main": "index.js",
-  "license": "MIT"
+  "license": "MIT",
+  "devDependencies": {
+    "babel-core": "^6.26.3",
+    "babel-loader": "^7.1.5",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-react": "^6.24.1",
+    "webpack": "^4.16.5",
+    "webpack-cli": "^3.1.0"
```

Comment:
There might be newer versions for these guys.

Reply:
I'd also try to get webpack dev server running.

Reply:
For webpack, what I have here are the latest versions:
https://github.com/webpack/webpack/releases
https://github.com/webpack/webpack-cli/releases

Reply:
I was considering also the `babel`s.

Reply:
Are you referring to https://www.npmjs.com/package/@babel/preset-env and https://www.npmjs.com/package/@babel/preset-react?

Reply:
👍